docs/data/pages.ts

```diff
@@ -125,6 +125,30 @@ const pages: MuiPage[] = [
       },
     ],
   },
+  {
+    pathname: '/toolpad/examples',
+    icon: DescriptionIcon,
+    children: [
+      // { pathname: '/toolpad/examples/admin-app' },
```

Comment:
We can get rid of any unused paths instead of leaving them in the comments

Reply:
This will come in handy soon enough. So I thought of keeping it.

Reply:
I think we can add it when it's needed